safe_transaction_service/safe_messages/utils.py

```diff
@@ -1,22 +1,36 @@
 from typing import Any
 
-from eth_account.messages import defunct_hash_message
+from eth_account.messages import encode_defunct
 from eth_typing import ChecksumAddress, Hash32
 from safe_eth.eth import get_auto_ethereum_client
-from safe_eth.eth.eip712 import eip712_encode_hash
+from safe_eth.eth.eip712 import eip712_encode
 from safe_eth.safe import Safe
 
 
-def get_hash_for_message(message: str | dict[str, Any]) -> Hash32:
+def encode_eip191_message(message: str) -> bytes:
```

Comment:
why is this needed?

Reply:
I need to encode the provided message somehow, but I don't want the hash, so I had to write this